### PR TITLE
Create new jobs for kind-1.19-sriov provider

### PR DESCRIPTION
**Pull request by ormergi**
Signed-off-by: Or Mergi <ormergi@redhat.com>

Following kubevirt/kubevirtci#621, we need to test it on CI environment 
as kind-1.19-sriov going to be new provider for SRIOV lanes.

This PR adds new kubvirtci-check-up and kubvirt-pull-e2e jobs for it.

### CODE REVIEW

**Review by fgimenez — APPROVED** [review]
/hold
/approve

Feel free to unhold when ready. 

As a side note, we will have soon the SR-IOV machines ready in the new cluster, these jobs should be set with `cluster: prow-workloads` instead of `cluster: phx-prow`. If the machines are ready before this PR is merged I'll drop a comment here.

**Comment by ormergi** [comment]
> /hold
> /approve
> 
> Feel free to unhold when ready.
>

Thank you :slightly_smiling_face: 

> As a side note, we will have soon the SR-IOV machines ready in the new cluster, these jobs should be set with `cluster: prow-workloads` instead of `cluster: phx-prow`. If the machines are ready before this PR is merged I'll drop a comment here.

Thanks for the heads up!
We need these jobs in order to continue with kubevirt/kubevirtci#621
it is prioritize because the current SRIOV provider runs k8s-1.17
The plan is to replace the sriov jobs that we use today with those that presented on this PR

/unhold

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml
Comment (xpivarc) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:952]: This should probably be false while we don't have it in kubevirt/kubevirt.

Comment (fgimenez) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:952]: Absolutely, I missed this setting, thanks for raising it! These jobs should replace the existing 1.17 lanes.

**Pull request merged**
